src/vs/workbench/contrib/webview/browser/overlayWebview.ts

```diff
@@ -207,6 +211,13 @@ export class OverlayWebview extends Disposable implements IOverlayWebview {
 				this._webview.value.setContextKeyService(this._scopedContextKeyService.value);
 			}
 
+			// Need to show the find widget again if we're supposed to retain context
+			// We previously hid find when we did this.release()
+			// https://github.com/microsoft/vscode/issues/157424
+			if (this._options.retainContextWhenHidden) {
+				this._webview.value.showFind();
```

Comment:
What if the find widget wasn't previously showing? Will this cause it to show again?

Reply:
Thanks for the callout!  That does indeed need to be handled.

I'll start looking into a way to handle that cleanly.  Preferably the solution would avoid any more state variables, but it may end up being necessary.  Just depends how everything is currently implemented so I'll dig into it a bit more.